core/sql/regress/executor/EXPECTED130

```diff
@@ -124,7 +134,16 @@ inserted row13
 
 --- 3 row(s) selected.
 >>
->>
+>>values (filetolob('./myfile.txt'));
+
+*** ERROR[8442] Unable to access ExpLOB interface. Call to ExpLOB returned error LOB_PTR_ERROR(532). Error detail 0.
```

Comment:
That sounds like a static semantic restriction. Shouldn't the binder catch that? I'm wondering if there are other contexts where the function should be forbidden. For example, inside a CAST, or SUBSTRING, or a column reference in a GROUP BY clause?